src/coverlet.msbuild.tasks/coverlet.msbuild.targets

```diff
@@ -66,7 +66,9 @@
       ThresholdType="$(ThresholdType)"
       ThresholdStat="$(ThresholdStat)"
       InstrumenterState="$(InstrumenterState)"
-      CoverletMultiTargetFrameworksCurrentTFM="$(_coverletMultiTargetFrameworksCurrentTFM)" />
+      CoverletMultiTargetFrameworksCurrentTFM="$(_coverletMultiTargetFrameworksCurrentTFM)">
+      <Output TaskParameter="ReportItems" ItemName="CoverletReport" />
```

Comment:
I thought about it too but decided to go with the singular spelling because basically all MSBuild [item elements](https://docs.microsoft.com/en-us/visualstudio/msbuild/item-element-msbuild) have a singular name: `ProjectReference`, `PackageReference`, `None`, `Compile`, `EmbeddedResource`, `Service` etc.

Reply:
ok I agree